requirements/test-requirements.in

```diff
@@ -6,7 +6,7 @@ git+git://github.com/millerdev/django-nose@fast-first-1.4.4#egg=django-nose
 fakecouch==0.0.15
 nose==1.3.7
 nose-exclude==0.5.0
-pip-tools>=4.0.0
+pip-tools>4.0.0
```

Comment:
I think the right answer is whatever is going to cause this to happen the least often in the future. Since this only gets updated when we target it to, it'll likely be tested any time it's updated, so I think the ceiling restriction is unlikely to save significant time. Testing that is fairly easy, since pip-tools is only used to generate these requirements files. On the other hand, having an upper limit may delay us in getting a fix similar to the one in this PR. That said, I think it's a fairly weak argument. What's your pitch for `>4.2.0,<5.0.0`?

Reply:
> What's your pitch for >4.2.0,<5.0.0?

(I should have actually written >=4.2.0) At the time I was thinking that our tests only passed with 4.2.0 since that's what we're upgrading to. I suppose if our tests are only failing with 4.0.0 then that's fine.

the <5 was so that we could prevent future failures, but if our tests are fragile enough (or the library puts out bad breaking changes) to break between 4.0.0 and 4.0.1 or 4.1.0 then that wouldn't really help